examples/bitmap_operation/example_bitmap_operation.cpp

```diff
@@ -24,17 +24,18 @@ int main()
         // Second way to do
         method2();
     }
-    catch( const imageException & ex ) {
-        // uh-oh, something went wrong!
+    catch( imageException & ex ) { // uh-oh, something went wrong!
```

Comment:
Please remove this catch block from each file. As it's mentioned in the issue we replace `imageException` by `std::exception`  in catch block. `imageException` class is inherited from `std::exception` base class so catching base class exception would catching inherited (class) exception too.